inc/functions.php

```diff
@@ -884,10 +884,12 @@ function get_child_list($fid)
 /**
  * Produce a friendly error message page
  *
- * @param string $error The error message to be shown
+ * @param string $error_message The error message to be shown
  * @param string $title The title of the message shown in the title of the page and the error table
+ * @param string $error_page Optional additional error page
+ * @param int $status_code HTTP status code to send with the error page
  */
-function error($error_message = "", $title = "", $error_page = "")
+function error($error_message = "", $title = "", $error_page = "", $status_code = 200)
```

Comment:
Regardless of use implementing wide type hints, I think we should start using them for new or updated code like this line.

Also, `never` as the return type declaration, although I understand if you decide not to as you are not adding a new function here.